### DISCUSSION TITLE
Display field-name in single form

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#18919] [Feature Requests]
### Summary

Fields can be named in the settings. Sometimes a field has a different display-name for the form than it has in the database (and therewith in the API). To not always have ti switch between the different pages it would be great if admins could see the original name inside the forms.

### Basic Example

The original field-name could be either displayed in the existing dropown-menu or in a new Meta / Info-Menu:

![Bildschirmfoto 2023-06-16 um 13 12 19](https://github.com/directus/directus/assets/46492597/0d2d45ec-8f41-4d03-9d73-4f223eaa64fb)

When clicking on the name, it should be copied.

### Motivation

Find the original field-name from inside a item collection.

### Detailed Design

/

### Must Have:
- Display the name in the field-context

### Should Have:
- Copy to clipboard on click on the name
- Link to the Field-settings

### Drawbacks

Other non-admin roles (devs) could probably need this too, but as an MVP this could be added for admins only.

### Alternatives

/

### Adoption Strategy

/

**Comment by emahuni** [comment]
I think this is the same as #13277 though this not only referring to tables only, the feature is basically the same thing.

**Reply by Dominic-Marcelino** [reply]
Not really 🤔 #13277  is about modifying the displayed column names in layouts. What I refer to is being able to see the original db field name inside the items form
